core/src/main/java/discord4j/core/object/entity/ForumThreadMessage.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J. If not, see <http://www.gnu.org/licenses/>.
+ */
+package discord4j.core.object.entity;
+
+import discord4j.common.util.Snowflake;
+import discord4j.core.GatewayDiscordClient;
+import discord4j.core.object.DiscordObject;
+import discord4j.core.object.Embed;
+import discord4j.core.object.component.MessageComponent;
+import discord4j.discordjson.json.ForumThreadMessageParamsData;
+import discord4j.rest.util.AllowedMentions;
+
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * Entity representing thread message
+ */
+public final class ForumThreadMessage implements DiscordObject {
```

Comment:
most for know..

you cannot edit/delete this? i mean this represents the first message not? then you cannot remove or edit or any?